packages/web-app-files/src/mixins/spaces/actions/setReadme.js

```diff
@@ -48,26 +47,26 @@ export default {
     }
   },
   methods: {
-    ...mapMutations('Files', ['UPDATE_RESOURCE_FIELD']),
+    ...mapMutations('Files', ['UPDATE_SPACE_FIELD']),
     ...mapActions(['showMessage']),
     async $_setSpaceReadme_trigger({ resources }) {
-      const space = this.currentFolder
-
       try {
+        const storageId = this.$route.params.storageId
+        const space = this.spaces.find((s) => s.id === storageId)
```

Comment:
The `space` was set in the folder loader before, see `loaderProject.ts` line 77. Can/should we remove that from the folder loader then?